gluestick/reader.py

```diff
@@ -0,0 +1,167 @@
+import os
+import json
+import pandas as pd
+import pyarrow.parquet as pq
+
+class Reader:
```

Comment:
Was this just a full copy paste moving it out of `etl_utils` or did you change anything?

Reply:
yes full copy paste, no changes at all